src/css/theme/_base.scss

```diff
@@ -15,6 +15,27 @@
 @import 'stats';
 @import 'toggle';
 
+.ais-clear-all,
+.ais-current-refined-values,
+.ais-hierarchical-menu,
+.ais-menu,
+.ais-pagination,
+.ais-price-ranges,
+.ais-range-slider,
+.ais-refinement-list,
+.ais-search-box,
+.ais-star-rating,
+.ais-stats,
+.ais-toggle {
```

Comment:
It's css 2 actually :) https://developer.mozilla.org/en/docs/Web/CSS/Attribute_selectors

I think the performance hit of this is negligible, since most sites have border box on *